### PR TITLE
feat: add notes about changes with Pulse

### PR DESCRIPTION
**Pull request by ankur-arch**
With the new Pulse architecture rolling out, we have to make specific changes to the docs. Please read [this](https://www.notion.so/prismaio/Developer-Connections-Tasks-400641334c214c9ca7000a721cf339f7?p=4de1fae767214293aae806683474652c&pm=s) doc for more details.

### CODE REVIEW

File: content/400-pulse/200-getting-started.mdx [hunk 1]
```diff
@@ -58,6 +59,8 @@ ALTER SYSTEM SET wal_keep_size = 2048;
 
 ### Provider specific configuration
 
+To learn about the database providers that support Pulse, visit [here](/pulse/faq#what-database-providers-are-supported-with-pulse).
```
Comment (nikolasburk) [inline content/400-pulse/200-getting-started.mdx]:
```suggestion
To learn about the database providers that Pulse supports, visit [here](/pulse/faq#what-database-providers-are-supported-with-pulse).
```

File: content/400-pulse/200-getting-started.mdx [hunk 2]
```diff
@@ -113,6 +116,10 @@ SELECT pg_reload_conf();
 
 </details>
 
+##### ssl mode
```
Comment (nikolasburk) [inline content/400-pulse/200-getting-started.mdx]: either uppercase for proper spelling or write in backticks / code formatting and one word (`sslmode`) to indicate you're refering to the setting in code.

```suggestion
##### SSL mode
```

File: content/400-pulse/600-faq.mdx [hunk 3]
```diff
@@ -37,3 +37,12 @@ Pulse primarily interfaces with a database's write-ahead log (Postgres) or binlo
 Pulse’s database compatibility list will be different from Prisma Client's database compatibility list. If you want to use Pulse with an existing Prisma application, please verify the database and hosting provider are supported by Pulse.
 
 </Admonition>
+
+## What database providers are supported with Pulse?
+
+| Provider | Support  | Notes                                                                                                                                                                            |
```
Comment (nikolasburk) [inline content/400-pulse/600-faq.mdx:43]: Nice table 👌

**Review by nikolasburk — CHANGES_REQUESTED** [review]
Found one typo and made some suggestions but overall lgtm 👍

File: content/400-pulse/600-faq.mdx (hunk 3 above)
Comment (ankur-arch) [reply content/400-pulse/600-faq.mdx:43]: Thank you 🙇

File: content/400-pulse/200-getting-started.mdx (hunk 2 above)
Comment (ankur-arch) [reply content/400-pulse/200-getting-started.mdx]: Kept the upper case notation as a suggestion!

File: content/400-pulse/100-what-is-pulse.mdx [hunk 4]
```diff
@@ -10,8 +10,6 @@ toc: true
 
 [Pulse](https://www.prisma.io/data-platform/pulse) is a managed [change data capture (CDC)](https://en.wikipedia.org/wiki/Change_data_capture) service that captures change events from your database and delivers them instantly to your applications. With Pulse, you can quickly build real-time applications in a type-safe manner using [Prisma Client](/orm/prisma-client).
 
-![Pulse usage diagram.](../doc-images/data-platform/platform/pulse//diagram.png)
```
Comment (nikolasburk) [inline content/400-pulse/100-what-is-pulse.mdx:13]: btw, maybe we could create a design task to have an updated version of Pulse's architecture here? same for Accelerate actually? I think having technical diagrams on the "What is X" page of the PDP products could be great for quickly telling people what these are / how they work in a visual way.

File: content/400-pulse/200-getting-started.mdx [hunk 5]
```diff
@@ -13,7 +13,7 @@ toc: true
 To participate in Pulse's Early Access program, you need to meet the following prerequisites:
 
 - A GitHub account.
-- A project that uses [Prisma Client](/orm/prisma-client) `4.16.1` or higher.
+- Pulse requires [Prisma Client](/orm/prisma-client) version `4.16.1` or higher and [@prisma/extension-pulse](https://www.npmjs.com/package/@prisma/extension-pulse) version `v0.2.0` or higher.
```
Comment (nikolasburk) [inline content/400-pulse/200-getting-started.mdx]:
```suggestion
- Pulse requires [Prisma Client](/orm/prisma-client) version `4.16.1` or higher and [`@prisma/extension-pulse`](https://www.npmjs.com/package/@prisma/extension-pulse) version `v0.2.0` or higher.
```

**Review by nikolasburk — APPROVED** [review]

File: content/400-pulse/100-what-is-pulse.mdx (hunk 4 above)
Comment (ankur-arch) [reply content/400-pulse/100-what-is-pulse.mdx:13]: Created a design task [here](https://www.notion.so/prismaio/Developer-Connections-Tasks-400641334c214c9ca7000a721cf339f7?p=07a69824b4cc44d48da96285411da2dc&pm=s). 

I'll handle that in a separate PR as that falls outside the scope of this PR.

File: content/400-pulse/200-getting-started.mdx [hunk 6]
```diff
@@ -23,6 +23,7 @@ You will also need a database with the following configurations:
 - Ensure your database is publicly accessible.
 - [Set the `wal_level` setting in PostgreSQL to `logical`](/pulse/getting-started#wal_level).
```
Comment (grpatel) [inline content/400-pulse/200-getting-started.mdx:24]: Is this line supposed to be a link? It links me to the same page

File: content/400-pulse/200-getting-started.mdx [hunk 7]
```diff
@@ -58,6 +59,8 @@ ALTER SYSTEM SET wal_keep_size = 2048;
 
 ### Provider specific configuration
 
+To learn about the database providers that Pulse supports, visit [here](/pulse/faq#what-database-providers-are-supported-with-pulse).
+
 #### Railway
 
 [Railway.app](https://railway.app) offers an excellent [templates feature](https://railway.app/templates). If you wish to quickly start with Pulse, you can use either of two templates:
```
Comment (grpatel) [inline content/400-pulse/200-getting-started.mdx:66]: I believe the restart button for the Railway template is now under the "Deployments" tab right?

**Review by grpatel — COMMENTED** [review]
Looks good to me!

File: content/400-pulse/200-getting-started.mdx (hunk 7 above)
Comment (ankur-arch) [reply content/400-pulse/200-getting-started.mdx:66]: That's a great catch @grpatel 💯

File: content/400-pulse/200-getting-started.mdx (hunk 6 above)
Comment (ankur-arch) [reply content/400-pulse/200-getting-started.mdx:24]: It links to the section heading `wal_level, it's supposed to take you there. Seems broken.

File: content/400-pulse/600-faq.mdx [hunk 8]
```diff
@@ -37,3 +37,12 @@ Pulse primarily interfaces with a database's write-ahead log (Postgres) or binlo
...
+| -------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Railway  | ✅       | Requires [`sslmode=disable`](/orm/overview/databases/postgresql#configuring-an-ssl-connection) due to the use of a self-signed certificate for database connections by Railway.  |
+| Supabase | ✅       | Requires [`sslmode=disable`](/orm/overview/databases/postgresql#configuring-an-ssl-connection) due to the use of a self-signed certificate for database connections by Supabase. |
+| Neon     | _Coming_ | Neon doesn't allow superuser access, hence, currently not supported with Pulse.                                                                                                  |
```
Comment (luanvdw) [inline content/400-pulse/600-faq.mdx]:
```suggestion
| Neon     | _Coming_ | Support details coming soon.                                                                                                 |
```

**Review by luanvdw — COMMENTED** [review]
@ankur-arch G2G from my side, thank you!

Added a small copy change below. We can omit these exact details for now.

**Comment by mhessdev** [comment]
Looks good to me!

**Review by luanvdw — APPROVED** [review]
LGTM, thanks @ankur-arch

**Pull request merged**